MvvmCross/Platform/Tvos/Core/MvxApplicationDelegate.cs

```diff
@@ -3,14 +3,47 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using Foundation;
 using MvvmCross.Core;
+using MvvmCross.ViewModels;
 using UIKit;
 
 namespace MvvmCross.Platform.Tvos.Core
 {
-    public class MvxApplicationDelegate : UIApplicationDelegate
-                                          , IMvxApplicationDelegate
+    public abstract class MvxApplicationDelegate : UIApplicationDelegate, IMvxApplicationDelegate
     {
+        private MvxTvosSetup _setup;
+        protected MvxTvosSetup Setup
+        {
+            get
+            {
+                if (_setup == null)
+                    _setup = CreateSetup(this, Window);
```

Comment:
Can we make tvOS work the same way as Android and iOS? (use a `InitializePlatform` method instead of ctor arguments) It seems cleaner to me to unify all setup changes on the same PR

Reply:
Done - as per mac, can someone please test to make sure my changes work